src/main/java/at/hannibal2/skyhanni/config/features/MiscConfig.java

```diff
@@ -251,6 +251,12 @@ public static class HideArmor {
     @ConfigAccordionId(id = 11)
     public boolean estimatedIemValueArmor = true;
 
+    @Expose
+    @ConfigOption(name = "Show Estimated Chest Value", desc = "Will show the total values of the items in a chest and the most expensive ones.")
+    @ConfigEditorBoolean
+    @ConfigAccordionId(id = 11)
+    public boolean estimatedChestValue = false;
```

Comment:
please rename the value to `estimatedIemValueChest` to keep the same syntax as the other elements in this category